pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -85,6 +86,10 @@ const (
 	// load balancer.
 	gcpLBTypeAnnotation = "cloud.google.com/load-balancer-type"
 
+	// GCPGlobalAccessAnnotation is the annotation used on an internal load balancer service
+	// to enable the GCP Global Access feature.
+	GCPGlobalAccessAnnotation = "networking.gke.io/internal-load-balancer-allow-global-access"
```

Comment:
I would suggest moving this to `api/operator/v1/types_ingress.go`, along with the other new GCP consts.

Reply:
Typically we don't put implementation specific constants in the ingress controller API. I think it makes the most sense to define the annotation constant here alongside all of the other load balancer service annotations.